cunumeric/module.py

```diff
@@ -994,7 +994,10 @@ def ndim(a: ndarray) -> int:
     --------
     Multiple GPUs, Multiple CPUs
     """
-    return a.ndim
```

Comment:
Does the original code fail? I would expect it to work, since the `@add_boilerplate("a")` decorator above converts anything to a cunumeric.ndarray.

Reply:
It will raise error: " AttributeError: 'NoneType' object has no attribute 'ndim'", when inp is None in num.ndim(inp).

Reply:
I see, can you do it like this then?
```
return 0 if a is None else a.ndim
```